### PR TITLE
cross build with scala 2.12

### PR DESCRIPTION
**Pull request by kcaipq**

### CODE REVIEW

**Review by tomverran — APPROVED** [review]

**Comment by kcaipq** [comment]
@tomverran can you take another look?

File: build.sbt [hunk 1]
```diff
@@ -23,7 +24,6 @@ libraryDependencies ++= Seq(
   "io.circe" %% "circe-parser" % circeVersion,
   "com.typesafe.akka" %% "akka-http-testkit" % akkaHttpVersion % "test",
   "org.scalatest" %% "scalatest" % scalaTestVersion % "test"
-    exclude("org.scala-lang.modules", "scala-xml_2.11")
```
Comment (tomverran) [inline build.sbt:26]: I wonder why we did this

**Review by tomverran — APPROVED** [review]
👍

File: build.sbt (hunk 1 above)
Comment (kcaipq) [reply build.sbt:26]: oh I think I did it just to avoid the version conflicts

**Pull request merged**